healthy_plane_calculations.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from skbio.stats.ordination import OrdinationResults\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from plane import point_to_segment_distance\n",
+    "from itertools import product\n",
+    "\n",
+    "import seaborn as sns\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# clearning metadata from analysis\n",
+    "metadata = pd.read_csv('15093_analysis_mapping.txt.gz', compression='gzip', sep='\\t', dtype=str)\n",
+    "metadata.set_index('#SampleID', inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(0.828031 x) + (0.357078 y) + (-1.000000 z) + (0.035064) \n"
+     ]
+    }
+   ],
+   "source": [
+    "pcoa_fp = 'ordination.txt.gz'\n",
+    "pcoa = OrdinationResults.read(pcoa_fp, compression='gzip')\n",
+    "df = metadata.loc[pcoa.samples.index.tolist()]\n",
+    "hc_ids = df[df.diagnosis_simple == 'HC'].index.tolist()\n",
```

Comment:
Most of this can be replaced with a call to `distance_to_reference_plane`:

```python
metadata['distance_to_plane'] = distance_to_reference_plane(pcoa, metadata, 'HC', 'diagnosis_simple')
```